docs/source/markdown/podman-run.1.md

```diff
@@ -89,14 +89,15 @@ Set the cgroup namespace mode for the container.
 
 If the host uses cgroups v1, the default is set to **host**.  On cgroups v2, the default is **private**.
 
-**--cgroups**=**enabled**|**disabled**|**no-conmon**
+**--cgroups**=**enabled**|**disabled**|**no-conmon**|**split**
 
 Determines whether the container will create CGroups.
 
 Default is **enabled**. The **disabled** option will force the container
```

Comment:
Can we add that "enabled" created a new cgroup under the cgroup parent, to distinguish it from "split" which reuses the current cgroup?

Reply:
added to the last version